cmd/template/cluster/testdata/run_template_cluster_capa_2.golden

```diff
@@ -12,6 +13,8 @@ data:
           httpsProxy: https://internal-a1c90e5331e124481a14fb7ad80ae8eb-1778512673.eu-west-2.elb.amazonaws.com:4000
           noProxy: test-domain.com
         subnets:
+        - cidrBlocks: []
```

Comment:
This is a private cluster and it should not have any public subnets (and empty `cidrBlocks` is wrong)